docs/Installing.asciidoc

```diff
@@ -99,13 +135,63 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database
+
+Older versions of openQA, like the one distributed on Leap 42.3, had support for
```

Comment:
s/Leap 42.3/openSUSE Leap 42.3/

I would rephrase the section to start with the current version describing postgreSQL and then mention older versions as the exception